builder/vsphere/common/step_export.go

```diff
@@ -105,6 +109,16 @@ type ExportConfig struct {
 	//   }
 	// ```
 	Options []string `mapstructure:"options"`
+	// The format to export the virtual machine image to. Defaults to `ovf`.
+	// Available options include `ovf` and `ova`.
+	//
+	// When set to `ova`, the image is first exported using Open Virtualization Format (`.ovf`) and
+	// then converted to an Open Virtualization Archive (`.ova`) using the VMware [Open Virtualization Format Tool](https://developer.vmware.com/web/tool/ovf/)
+	// (ovftool). The intermediate files are removed after the conversion.
+	//
+	// ~> **Note:** To use the `ova` format option, VMware ovftool must be installed on the Packer host
+	// and accessible in either the system `PATH` or the user's `PATH`.
+	Format string `mapstructure:"format"`
```

Comment:
Very nitpicky and feel free to push against that, but I'd think renaming the option as `output_format` may be better to make it clear we're talking about the file format of the output artifact, and not a random format string.

Reply:
I actually like this idea. 😃

Reply:
Resolved in 97a2527.